packages/@glimmer/wire-format/lib/elements.ts

```diff
@@ -0,0 +1,60 @@
+export enum Tags {
```

Comment:
@chadhietala Isn't there a plan to provide the compiler with a list of known component names?